helpers/http-helper.go

```diff
@@ -0,0 +1,37 @@
+package helpers
```

Comment:
Please put it in the aha package (where you use it). Stuff is either in cmd/... or pkg/..